.github/workflows/main.yml

```diff
@@ -4,7 +4,7 @@ name: CI
 
 # Controls when the action will run. Triggers the workflow on push or pull request
 # events but only for the master branch
-on: [push, pull_request]
+on: [push, pull_request, workflow_dispatch]
 
```

Comment:
``dispatch`` is for triggering manually, right? I remember seeing that somewhere.

Reply:
Yep, see https://github.blog/changelog/2020-07-06-github-actions-manual-triggers-with-workflow_dispatch/ , added so if someone has a problem then we can trigger workflow without having to make a commit or PR.